stable/goldilocks/templates/controller-deployment.yaml

```diff
@@ -41,8 +41,10 @@ spec:
             {{- range $name, $value := .Values.controller.flags }}
             - --{{ $name }}={{ $value }}
             {{- end }}
+        {{- if .Values.controller.securityContext }}
```

Comment:
Just realizing now that we already had the securityContext in here. Does setting `controller.securityContext: {}` not effectively do this in the existing version of the chart?

Reply:
It doesn't - Helm will deep merge two hashes, so everything from the default values.yaml will still be there

Reply:
Interesting. I hadn't thought about that. 👍🏼